storybook/figmalinkssource.cpp

```diff
@@ -38,6 +38,74 @@ FigmaLinks* FigmaLinksSource::getFigmaLinks() const
     return m_figmaLinks;
 }
 
+void FigmaLinksSource::remove(const QString &key, const QList<int> &indexes)
```

Comment:
On a second thought, QML knows about the QModelIndex type; you could simplify both this and QML code as well using it directly

Reply:
Yeah, but I'm not even removing it from model. I just remove it from representation stored in `FigmaLinksSource` and save to file. The rest is done automatically (updating the model) via reloading it from file again.

Reply:
Ok cool, was just thinking loud :)